src/frontend/src/lib/api/signer.api.ts

```diff
@@ -30,6 +32,18 @@ export const getEthAddress = async ({ identity }: CommonParams): Promise<EthAddr
 	return getEthAddress();
 };
 
+export const updateBtcBalance = async ({
```

Comment:
Just noticing it now, sorry did not picked it on first review, why is it prefixed with `update...`? why not just `btcBalance` or `getBtcBalance`?

Reply:
The idea was to highlight that it's a certified balance (on the contrary to the `query` balance method available in `bitcoin.api`). However, now that I think about it again, `getBtcBalance` makes much more sense since the "certified" part is already implied by the use of the signer canister. Renaming.